README.md

```diff
@@ -304,6 +301,35 @@ It will also exclude any files with the extension `.spec.js`.
 **Note:** Be wary of automatic OS glob expansion when specifying include/exclude globs with the CLI.
 To prevent this, wrap each glob in single quotes.
 
+### Including files within `node_modules`
+
+We always add `**/node_modules/**` to the exclude list, even if not specified in the config.
+You can override this by setting `--exclude-node-modules=false`.
+
+For example, the following config will instrument everything in `lib/` and `node_modules/@my-org/`, except for `node_modules/@my-org/something/unwanted.js`.
+All other files in `node_modules` are filtered out by the include rules.
+The exclude rules prevent nyc instrumenting test files and `node_modules/@my-org/something/unwanted.js`
+
+```json
+{
+  "nyc": {
+    "all": true,
+    "include": [
+      "lib/**",
+      "node_modules/@my-org/**"
+    ],
+    "exclude": [
+      "node_modules/@my-org/something/unwanted.js",
+      "**/test/**"
+    ],
+    excludeNodeModules: false
```

Comment:
Missing quotes for JSON property name.

Reply:
And for some reason I thought the big red squiggly line was about spelling...